package.json

```diff
@@ -44,13 +46,18 @@
   "engines": {
     "node": ">=18.0.0"
   },
+  "exports": {
+    "import": "./dist/index.js",
+    "require": "./cjs/index.js"
+  },
   "dependencies": {
     "@types/json-schema": "^7.0.15",
     "commander": "^12.0.0",
     "glob": "^10.3.12",
     "json5": "^2.2.3",
     "normalize-path": "^3.0.0",
     "safe-stable-stringify": "^2.4.3",
+    "tslib": "^2.6.2",
```

Comment:
Why is this change needed?

Reply:
I don't think we need to reduce build size in the same pull request.

Reply:
`tslib` one of the most downloaded packages in npm. Everyone who installs one of its 120.000 dependents packages will have it already installed in their machine. 

As after this PR this package size will almost double (two js files, two js sourcemaps and two d.ts for each source file), enabling this tsconfig option could reduce the package download time and improve the CJS codebase in general.

Without `tslib` & `importHelpers`:

```js
"use strict";
var __createBinding = (this && this.__createBinding) || (Object.create ? (function(o, m, k, k2) {
    if (k2 === undefined) k2 = k;
    var desc = Object.getOwnPropertyDescriptor(m, k);
    if (!desc || ("get" in desc ? !m.__esModule : desc.writable || desc.configurable)) {
      desc = { enumerable: true, get: function() { return m[k]; } };
    }
    Object.defineProperty(o, k2, desc);
}) : (function(o, m, k, k2) {
    if (k2 === undefined) k2 = k;
    o[k2] = m[k];
}));
var __exportStar = (this && this.__exportStar) || function(m, exports) {
    for (var p in m) if (p !== "default" && !Object.prototype.hasOwnProperty.call(exports, p)) __createBinding(exports, m, p);
};
var __importDefault = (this && this.__importDefault) || function (mod) {
    return (mod && mod.__esModule) ? mod : { "default": mod };
};
Object.defineProperty(exports, "__esModule", { value: true });
exports.ts = void 0;
__exportStar(require("./src/Error/BaseError.js"), exports);
```

With them:

```js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.ts = void 0;
const tslib_1 = require("tslib");
tslib_1.__exportStar(require("./src/Error/BaseError.js"), exports);
```

And it gets repeated with almost every commonsjs javascript file. 

This change does not affect any ESM user.

Reply:
Would it help for esm users? If so, let's move the import helpers to esm as well. Or is it not needed since esm has all the things we need? I'd still put it into the main tsconfig just because in the future we might use it.

Reply:
Most of the things `tslib` helps is with import/export, but sure! It might help with some usecases.